src/components/tx/modals/NewTxModal/styles.module.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: var(--space-4) !important;
+}
+
+.redCircle {
+  border: 3px solid var(--color-error-main);
+}
+
+.circle {
+  border: 2px solid var(--color-border-light);
+}
+
+.circle,
+.redCircle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+}
+
+.stepper {
+  padding-top: var(--space-3);
+  gap: var(--space-2);
+}
+
+.stepper :global .MuiStepConnector-root {
+  left: calc(-50% + 14px);
+  right: calc(50% + 28px);
```

Comment:
nit: Its also possible to do this via transform which is a bit clearer regarding the magic numbers.

```
left: -8px;
transform: translateX(-50%);
width: calc(100% - 32px);
```

By changing the second parameter in width its possible to adjust the size of the connector without having to change any other properties. This solution only works if all columns have the same size though.